main.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // Config contains the application configuration parameters
 type Config struct {
-	BaseDir      string   `envconfig:"BASE_DIR" default:"."`
```

Comment:
I'd rather leave this alone in the code so you can actually test it locally more easily. We should really check this in the startup script. Otherwise if you don't specify it, S6 will just restart it forever.